new-lamassu-admin/src/components/inputs/base/Dropdown.js

```diff
@@ -0,0 +1,29 @@
+import FormControl from '@material-ui/core/FormControl'
+import InputLabel from '@material-ui/core/InputLabel'
+import MenuItem from '@material-ui/core/MenuItem'
+import Select from '@material-ui/core/Select'
+import classnames from 'classnames'
+import React from 'react'
+const Dropdown = ({ label, name, options, onChange, value, className }) => {
+  return (
+    <FormControl className={classnames(className)}>
+      <InputLabel>{label}</InputLabel>
+      <Select
+        autoWidth={true}
+        labelId={label}
+        id={name}
+        value={value}
+        onChange={onChange}>
+        {options.map((option, index) => {
+          return (
+            <MenuItem key={index} value={option.value}>
+              {option.display}
+            </MenuItem>
+          )
+        })}
```

Comment:
You can cut a bit of code here if you want. Either way, it's better to use `R.map`, instead of the default JS  higher-order functions.

```suggestion
        {options.map((option, index) => (
            <MenuItem key={index} value={option.value}>
              {option.display}
            </MenuItem>
          )
        )}
```

Reply:
Interesting, Ramda also does not support index on R.map
Making a new constant like mymap = R.addIndex(R.map) seems to work but addIndex has been removed from Ramda's docs and I don't wanna risk it.